2023-03-11 08:50:19,526 INFO    MainThread:7266 [wandb_setup.py:_flush():71] setting env: {'silent': 'true'}
2023-03-11 08:50:19,527 INFO    MainThread:7266 [wandb_setup.py:_flush():71] setting login settings: {}
2023-03-11 08:50:19,527 INFO    MainThread:7266 [wandb_init.py:_log_setup():371] Logging user logs to /workspace/wandb/run-20230311_085019-1o9gnzah/logs/debug.log
2023-03-11 08:50:19,527 INFO    MainThread:7266 [wandb_init.py:_log_setup():372] Logging internal logs to /workspace/wandb/run-20230311_085019-1o9gnzah/logs/debug-internal.log
2023-03-11 08:50:19,527 INFO    MainThread:7266 [wandb_init.py:_jupyter_setup():321] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7efdf53f44d0>
2023-03-11 08:50:19,528 INFO    MainThread:7266 [wandb_init.py:init():404] calling init triggers
2023-03-11 08:50:19,529 INFO    MainThread:7266 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {'CONTACT_IMG_DIR': '/workspace/input/contact_images', 'DEBUG': True, 'EXP_CATEGORY': 'DEBUG', 'EXP_NAME': 'DEBUG', 'INPUT_DIR': '/workspace/input', 'MODEL_DIR': '/workspace/output/DEBUG/model', 'OUTPUT_DIR': '/workspace/output', 'SAVED_CONTACT_CSV': '/workspace/input/Saved_contact_frames.csv', 'TRAIN_HELMET_CSV': '/workspace/input/train_baseline_helmets.csv', 'TRAIN_LABEL_CSV': '/workspace/input/train_labels.csv', 'TRAIN_TRACKING_CSV': '/workspace/input/train_player_tracking.csv', 'TRAIN_VIDEO_META_CSV': '/workspace/input/train_video_metadata.csv', 'TRAIN_VIDEO_NUM': 100, 'T_max': 10, 'VALID_VIDEO_NUM': 10, 'batch_size': 32, 'features': ['x_position_1', 'y_position_1', 'x_position_2', 'y_position_2', 'speed_1', 'distance_1', 'direction_1', 'orientation_1', 'acceleration_1', 'sa_1', 'speed_2', 'distance_2', 'direction_2', 'orientation_2', 'acceleration_2', 'sa_2', 'players_dis'], 'img_size': [224, 224], 'inp_channels': 3, 'kaggle': False, 'lr': 0.0001, 'masksize_helmet_ratio': 4, 'min_lr': 1e-08, 'model_name': 'swin_s3_tiny_224', 'n_epoch': 2, 'n_folds': 3, 'num_img_feature': 10, 'num_workers': 2, 'out_features': 20, 'pretrained': True, 'print_freq': 1000, 'random_seed': 21, 'roll_sum_window_size': 10, 'sample_num': 1000, 'train_folds': [0, 1, 2], 'wandb': True, 'weight_decay': 1e-06}
2023-03-11 08:50:19,529 INFO    MainThread:7266 [wandb_init.py:init():460] starting backend
2023-03-11 08:50:19,529 INFO    MainThread:7266 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2023-03-11 08:50:19,542 INFO    MainThread:7266 [backend.py:ensure_launched():216] starting backend process...
2023-03-11 08:50:19,553 INFO    MainThread:7266 [backend.py:ensure_launched():222] started backend process with pid: 7433
2023-03-11 08:50:19,554 INFO    MainThread:7266 [wandb_init.py:init():469] backend started and connected
2023-03-11 08:50:19,568 INFO    MainThread:7266 [wandb_run.py:_label_probe_notebook():911] probe notebook
2023-03-11 08:50:19,570 ERROR   MainThread:7266 [jupyter.py:attempt_kaggle_load_ipynb():279] Unable to load kaggle notebook
Traceback (most recent call last):
  File "/opt/conda/lib/python3.7/site-packages/wandb/jupyter.py", line 273, in attempt_kaggle_load_ipynb
    client = kaggle.UserSessionClient()
  File "/root/.local/lib/python3.7/site-packages/kaggle_session.py", line 14, in __init__
    self.web_client = KaggleWebClient()
  File "/root/.local/lib/python3.7/site-packages/kaggle_web_client.py", line 30, in __init__
    'A JWT Token is required to call Kaggle, '
kaggle_web_client.CredentialError: A JWT Token is required to call Kaggle, but none found in environment variable KAGGLE_USER_SECRETS_TOKEN
2023-03-11 08:50:19,570 INFO    MainThread:7266 [wandb_run.py:_label_probe_notebook():921] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-03-11 08:50:19,571 INFO    MainThread:7266 [wandb_init.py:init():533] updated telemetry
2023-03-11 08:50:19,636 INFO    MainThread:7266 [wandb_init.py:init():563] communicating current version
2023-03-11 08:50:20,379 INFO    MainThread:7266 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.13.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2023-03-11 08:50:20,380 INFO    MainThread:7266 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2023-03-11 08:50:20,705 INFO    MainThread:7266 [wandb_init.py:init():606] starting run threads in backend
2023-03-11 08:50:25,707 INFO    MainThread:7266 [wandb_run.py:_console_start():1810] atexit reg
2023-03-11 08:50:25,710 INFO    MainThread:7266 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.WRAP
2023-03-11 08:50:25,711 INFO    MainThread:7266 [wandb_run.py:_redirect():1721] Wrapping output streams.
2023-03-11 08:50:25,712 INFO    MainThread:7266 [wandb_run.py:_redirect():1745] Redirects installed.
2023-03-11 08:50:25,713 INFO    MainThread:7266 [wandb_init.py:init():633] run started, returning control to user process
2023-03-11 08:50:25,713 INFO    MainThread:7266 [wandb_init.py:_pause_backend():284] pausing backend
2023-03-11 08:50:25,714 INFO    MainThread:7266 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2023-03-11 08:50:25,860 INFO    MainThread:7266 [wandb_init.py:_resume_backend():293] resuming backend
2023-03-11 08:50:25,863 INFO    MainThread:7266 [wandb_init.py:_pause_backend():284] pausing backend
2023-03-11 08:50:25,864 INFO    MainThread:7266 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2023-03-11 08:50:26,029 INFO    MainThread:7266 [wandb_init.py:_resume_backend():293] resuming backend
2023-03-11 08:50:27,619 INFO    MainThread:7266 [wandb_init.py:_pause_backend():284] pausing backend
2023-03-11 08:50:27,620 INFO    MainThread:7266 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2023-03-11 08:50:27,744 INFO    MainThread:7266 [wandb_init.py:_resume_backend():293] resuming backend
2023-03-11 08:50:32,679 INFO    MainThread:7266 [wandb_init.py:_pause_backend():284] pausing backend
2023-03-11 08:50:32,679 INFO    MainThread:7266 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2023-03-11 08:50:32,793 INFO    MainThread:7266 [wandb_init.py:_resume_backend():293] resuming backend
2023-03-11 08:50:32,934 INFO    MainThread:7266 [wandb_init.py:_pause_backend():284] pausing backend
2023-03-11 08:50:32,935 INFO    MainThread:7266 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2023-03-11 08:50:33,044 INFO    MainThread:7266 [wandb_init.py:_resume_backend():293] resuming backend
2023-03-11 08:50:35,782 INFO    MainThread:7266 [wandb_init.py:_pause_backend():284] pausing backend
2023-03-11 08:50:35,782 INFO    MainThread:7266 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2023-03-11 08:50:35,902 INFO    MainThread:7266 [wandb_init.py:_resume_backend():293] resuming backend
2023-03-11 08:50:38,015 INFO    MainThread:7266 [wandb_init.py:_pause_backend():284] pausing backend
2023-03-11 08:50:38,016 INFO    MainThread:7266 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2023-03-11 08:50:38,103 INFO    MainThread:7266 [wandb_init.py:_resume_backend():293] resuming backend
2023-03-11 08:50:38,135 INFO    MainThread:7266 [wandb_init.py:_pause_backend():284] pausing backend
2023-03-11 08:50:38,140 INFO    MainThread:7266 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2023-03-11 08:50:38,220 INFO    MainThread:7266 [wandb_init.py:_resume_backend():293] resuming backend
2023-03-11 08:50:43,186 INFO    MainThread:7266 [wandb_init.py:_pause_backend():284] pausing backend
2023-03-11 08:50:43,186 INFO    MainThread:7266 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2023-03-11 08:51:01,784 INFO    MainThread:7266 [wandb_init.py:_resume_backend():293] resuming backend
2023-03-11 08:51:02,347 INFO    MainThread:7266 [wandb_init.py:_pause_backend():284] pausing backend
2023-03-11 08:51:02,348 INFO    MainThread:7266 [jupyter.py:save_ipynb():374] not saving jupyter notebook
